telegram-translation-shortcuts.user.js

```diff
@@ -156,10 +235,10 @@ function handleShortcut (e) {
   switch (e.key) {
     // Cycle bindings
     case 'l':
-      if (!e.ctrlKey) cycleBindings(true)
+      if (!e.ctrlKey) cycleBindings(true); else matchedKey = false
```

Comment:
Hm, I’ll probably refactor this whole part later, it got a bit too messy with the if-else clauses.

Reply:
(Don’t worry about it.)